boards/arm/nrf52_vbluno52/doc/nrf52_vbluno52.rst

```diff
@@ -0,0 +1,109 @@
+.. _nrf52_vbluno52:
+
+nRF52-VBLUno52
+##############
+
+Overview
+********
+
+Zephyr can use the nrf52_vbluno52 board configuration to run on the VBLUno52 board,
+a VNG Bluetooth Low Energy UNO using an nRF52832 ARM Cortex-M4F processor.
+
+.. figure:: img/nrf52_vbluno52_pinout.png
+     :align: center
+     :alt: nRF52 VBLUno52
+
+     nRF52_VBLUno52 board
+
+Hardware
+********
+
+The VBLUno52 board has two external oscillators. The frequency of
+the slow clock is 32.768 kHz. The frequency of the main clock
+is 64 MHz.
+
+Supported Features
```

Comment:
Just checking, while the board may support these features, are all these features supported by Zephyr?  (I'm not aware of any FOTA support in Zephyr)